packages/app-builder-lib/src/options/macOptions.ts

```diff
@@ -175,6 +175,23 @@ export interface MacConfiguration extends PlatformSpecificBuildOptions {
    * Specify the URL of the timestamp authority server
    */
   readonly timestamp?: string | null
+
+  /**
+   * Whether to merge ASAR files for different architectures or not.
+   *
+   * This option has no effect unless building for "universal" arch.
+   * @default false
+   */
+  readonly mergeASARs?: boolean
```

Comment:
What are your thoughts on just having this default true since it'll only be applicable when building for universal?

Reply:
I think this is alright, but there are two downsides:

- It will require a major version bump
- The build will fail for some users and they will need to populate `singleArchFiles` with something like `"singleArchFiles": "node_modules/{@signalapp/signal-client/prebuilds/**,ringrtc/build/**,sharp/**}"`

If we are fine with both - I can flip it for sure!

Reply:
Sounds good. I'm okay with both since we're currently in a v23-alpha state for our next major version bump. It also forces individuals building universal binaries to leverage the latest features of @electron/universal, which seems like a practice we'd always want to follow too

Reply:
I went ahead and flipped it :joy: